esmvaltool/utils/recipe_test_workflow/doc/source/user_guide/workflow.rst

```diff
@@ -41,6 +41,19 @@ The |RTW| performs the following steps:
   :Details:
      Runs for every metric defined in the workflow
 
+``compare``
+  :Description:
+     Runs the compare script in |ESMValTool| on the recipe previously run
+     in ``process`` and a known good output.
```

Comment:
Since you mention the compare script from ESMValTool in the `Executes` section, to avoid repetition, how about:
```suggestion
     Compares the output from the ``process`` job with a Known Good Output
     (KGO).
```